### DISCUSSION TITLE
open in new tab

### DISCUSSION THREAD

**Discussion by Art-Che** [AutomaApp/automa#1010] [Q&A]
Good afternoon! Tell me please. How to open links in a new tab?

`iframe:nth-child(1) |> div:nth-child(1) > div:nth-child(3) > div > div > div > a`

I need to open different links in the same menu each time.
I need an analogue of the context menu command: "Open in a new tab"

**Comment by Kholid060** [comment]
Because the link is inside the iframe block you need to use the Attribute Value block to get the URL, use `iframe:nth-child(1) |> div:nth-child(1) > div:nth-child(3) > div > div > div > a` as the selector, and input `href` in the attribute name text field. After that, add a new tab block and input `{{variables.url}}` as the URL.

![image](https://user-images.githubusercontent.com/22908993/206882942-b4c5f976-00bb-488e-9558-6dd22a5ff01b.png)
![image](https://user-images.githubusercontent.com/22908993/206882984-5ae7eeb4-d52b-4dac-8a98-a3bacf0212c4.png)

**Reply by Art-Che** [reply]
Earned! Thank you very much for the help!
